### ISSUE TITLE
🐛 Bug: Charset Normalizer v 2.0.4 conflicts

### ISSUE THREAD

**Issue by GemmaTuron** [ersilia-os/ersilia#408]
### Describe the bug.

The warning message is:
`RequestsDependencyWarning: urllib3 (1.25.11) or chardet (5.0.0)/charset_normalizer (2.0.4) doesn't match a supported version!  warnings.warn("urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported "`
In my system I don't get it when I have the following versions:
- requests: 2.28.1
- urllib3: 1.25.11
- chardet: 5.0.0
- charset_normalizer: 2.1.1

The conflict therefore seems to come from the charset_normalizer version.

### Describe the steps to reproduce the behavior

1. Activate ersilia conda environment
2. check the above mentioned packages versions
3. Try to fetch a model from the CLI and see if you get the warning error
4. Upgrade to the mentioned versions of the packages and fetch again
5. Warning should have disappeared

### Expected behavior.

Warning removed if charset_normalizer version is updated to 2.1.1
If this is confirmed we can set it as a requirement when installing ersilia

### Operating environment

Ubuntu 20.04.1 LTS

**Comment by GemmaTuron** [comment]
@femme-js, related to issue #338 can you do the test explained above? Check which versions of the packages you have and paste them here, upgrade them and try the fetching again see if the warning has disappeared
_I don't think this is what is causing the models not to be fetched, more on this on the relevant channels, but it would still be good to avid the warning_

**Comment by Femme-js** [comment]
Hi  @GemmaTuron ! Sure, let me get back to you after testing.

**Comment by Femme-js** [comment]
requests==2.28.1
urllib3==1.25.11
chardet==5.0.0
charset-normalizer==2.1.1


Hi @GemmaTuron 
I am already having the updated versions in my environment, the same as you mentioned. 

I have successfully fetched the eos92sw model. I would say this might not be the issue causing model fetch failure. As in my new log output on eos1579, the warning have disappeared. This is after I updated my pip and conda versions as mentioned in issue #338
